src/main/java/com/epam/mentoring/homeWork/module5/task2/StatisticsModule.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.mentoring.homeWork.module5.task2;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by : Volodymyr_Silitskyi
+ * Created at : 9/3/2018
+ */
+
+
+public class StatisticsModule {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DisplayInfoFromPerson.class);
+    private static final int HERO_PARENTS = 2;
+    private static List<Person> listOfPersons = new ArrayList<>();
+
+    public static void addPerson(Person person) {
+        listOfPersons.add(person);
+    }
+
+    public static void logStatistics() {
+        catOwnersCounter();
+        parentsHeroesCounter();
+    }
+
+    private static void catOwnersCounter() {
+        List catOwners = listOfPersons.stream()
```

Comment:
unused variable. Either remove it or return. If you decide to return it, then define List type `List<Person>`